includes/blocks/podcast/markup.php

```diff
@@ -0,0 +1,115 @@
+<?php
```

Comment:
This file will need some of the logic from above duplicated to prevent items from displaying if the value is unset. Eg for season number, running time etc.

Attached image is the front end view:

![Screen Shot 2024-05-08 at 10 07 39 am](https://github.com/10up/simple-podcasting/assets/519727/1d99383d-24d5-4e43-8465-d381d2482d19)